sdk/include/reactos/uxundoc.h

```diff
@@ -1,7 +1,33 @@
 #pragma once
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 typedef HANDLE HTHEMEFILE;
 
+typedef struct tagTMERRINFO
+{
+    UINT nID;
+    WCHAR szParam1[MAX_PATH];
+    WCHAR szParam2[MAX_PATH];
+    WCHAR szFile[MAX_PATH];
+    WCHAR szLine[MAX_PATH];
+    INT nLineNo;
+} TMERRINFO, *PTMERRINFO;
```

Comment:
This structure is not used by exported functions, so you could move it inside `dll/win32/uxtheme/uxthemep.h`.

Reply:
Moved.